demisto_sdk/commands/pre_commit/hooks/hook.py

```diff
@@ -80,7 +80,8 @@ def filter_files_matching_hook_config(
             and (not exclude_pattern or not re.search(exclude_pattern, str(file)))
         }  # only exclude if defined
 
-    def _get_property(self, name, default=None):
+    @staticmethod
+    def get_property(hook: dict, mode: str, name: str, default=None):
```

Comment:
Still need the default `arg`?

Reply:
It makes sense to have a default value if the property is not found. Sometimes we would like to set it to `True`, for instance when we will want to filter out hooks with the `community: false` property